wiki/Tournaments/UKCC/6/en.md

```diff
@@ -7,14 +7,796 @@ tags:
+| **ram ranch** | ![][flag_GB] [MyAngelZeroTwo](https://osu.ppy.sh/users/8337591), ![][flag_GB] [BurninLight](https://osu.ppy.sh/users/9473917) |
+| **Чики Брики** | ![][flag_GB] [Icedd](https://osu.ppy.sh/users/11146469), ![][flag_GB] [Dant2003](https://osu.ppy.sh/users/9227012) |
+
+## Podium
+
+This competition has come to an end and resulted in the following podium:
+
+| Placing | Player |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | **Question Mark** (![][flag_GB] [Best459](https://osu.ppy.sh/users/8185366), ![][flag_GB] [Bubbleman](https://osu.ppy.sh/users/5182050)) |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | **10man?** (![][flag_GB] [majoreh](https://osu.ppy.sh/users/7959222), ![][flag_GB] [Kingling](https://osu.ppy.sh/users/7010761)) |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | **Boom! Tetris for Benbow** (![][flag_GB] [TopSp1n](https://osu.ppy.sh/users/7657761), ![][flag_GB] [Apo11o](https://osu.ppy.sh/users/9558549)) |
+
+## Mappools
+
+### Grand Finals
+
+**[Download the mappack here! (147 MB)](https://mega.nz/#!K5hV2Kob!zmN-Tre4e1ilQt9YL0r3xw1exEE336uGYxYPHnWYuJk)**
+
+- NoMod
+  - [UNDEAD CORPORATION - Yoru Naku Usagi wa Yume o Miru (MaridiuS) [Madness]](https://osu.ppy.sh/beatmapsets/695743#osu/1474012)
+  - [xi remixed by cosMo@bousouP - FREEDOM DiVE [METAL DIMENSIONS] (Cherry Blossom) [Extra]](https://osu.ppy.sh/beatmapsets/905158#osu/1907933)
+  - [Cranky - Ran (pishifat) [Extra]](https://osu.ppy.sh/beatmapsets/751772#osu/1582585)
+  - [Camellia - Towards the Horizon (Mir) [Adventure Time]](https://osu.ppy.sh/beatmapsets/714454#osu/1509915)
+  - [System of a Down - Vicinity of Obscenity (Larto) [Impossible]](https://osu.ppy.sh/beatmapsets/13768#osu/50712)
+  - [Krewella x Diskord - Beggars (Consouls Remix) (NeilPerry) [Neil x Ameth's Conflux]](https://osu.ppy.sh/beatmapsets/881753#osu/1843575)
+- Hidden
+  - [Renard - Banned Forever (Blue Dragon) [Lesjuh]](https://osu.ppy.sh/beatmapsets/16349#osu/64266)
+  - [96Neko - Uso No Hibana (Depths) [Corrupt Emotions]](https://osu.ppy.sh/beatmapsets/578574#osu/1225022)
+  - [M2U - Velocity (M2Usonek) [Slider Velocity]](https://osu.ppy.sh/beatmapsets/891581#osu/1863869)
+- HardRock
+  - [Hiroyuki Sawano - BLOWIN' (ktgster) [Challenge]](https://osu.ppy.sh/beatmapsets/560165#osu/1184609)
+  - [HujuniseikouyuuP - MISTAKE (val0108) [Ms. 0108]](https://osu.ppy.sh/beatmapsets/105245#osu/276366)
+  - [Shounen Radio - neu (Philippines) [Platinum]](https://osu.ppy.sh/beatmapsets/58422#osu/179070)
+- DoubleTime
+  - [Touyama Nao & Yamazaki Haruka - Ringo Ribbon (Fycho) [Insane]](https://osu.ppy.sh/beatmapsets/114490#osu/296435)
+  - [twiddy - Live from the Bowling Alley (squirrelpascals) [Collab Insane]](https://osu.ppy.sh/beatmapsets/873671#osu/1826146)
+  - [Harry - Earth Day (Moway) [Earth Day]](https://osu.ppy.sh/beatmapsets/65228#osu/195502)
+- FreeMod
+  - Blacklolita - Praeteritum \[Anachronism\]
```

Comment:
if it wasn't omitted on purpose
```suggestion
  - [Blacklolita - Praeteritum (JeZag) [Anachronism]](https://osu.ppy.sh/beatmapsets/651006#osu/2406944)
```

Reply:
Resolved with https://github.com/ppy/osu-wiki/pull/6341/commits/682662be8ff6091e687d573febf751a9eece6010